src/pages/AddRecipe.js

```diff
@@ -1,11 +1,78 @@
-import React from "react"
+import React, { useState } from "react"
+import { Button, Form, FormGroup, Label, Input } from "reactstrap"
+import { useNavigate } from "react-router-dom"
+
+const AddRecipe = ({ createRecipe, currentUser }) => {
+  const navigate = useNavigate()
+  const [myRecipe, SetMyRecipe] = useState({
```

Comment:
The casing best practice for hooks is using camelCasing. I would recommend updating SetMyRecipe to setMyRecipe